M2/Macaulay2/packages/VersalDeformations.m2

```diff
@@ -480,13 +840,15 @@ document {
      Headline => "calculating versal deformations and local Hilbert schemes",
      PARA{
      "This package provides tools for calculating tangent and obstruction spaces as well as
-     power series solutions for deformation problems involving isolated singularities and projective schemes, as well as deformations of modules."},
+     power series solutions for deformation problems involving isolated singularities and projective schemes, as well as deformations of modules.
+As of version 4.0, the package also computes nested versal deformations for pairs consisting of a scheme and subscheme, or module and a quotient module."},
  
      
      
     PARA{}, "A basic description of the package's approach to deformation problems can
-    be found at the documentation node for ",TO versalDeformation,". 
-    For details and mathematical background see ",
+    be found at the documentation node for ",TO versalDeformation,". A discussion
+    about nested deformations can be found at the documenation node for ",TO NestedDeformation,". 
```

Comment:
One of the tests is failing because of a typo:

```suggestion
    about nested deformations can be found at the documentation node for ",TO NestedDeformation,". 
```